https://issues.redhat.com/browse/ACM-18544 KI for subscription app (#7637)
open-cluster-management/rhacm-docs: release_notes/known_issues_application.adoc

See the following known issues for the Application lifecycle component.

> [#subscription-application-incorrect-warning]

Reviewer: @fxiang1 , should we write (Deprecated) behind `subscription application,` since this is announced this release and a pretty significant deprecation for the product. Typically we would do that.

Reviewer 2: Yes, if that's the best practice then let's add the deprecated text.

== Subscription application displays incorrect warning message
//2.13:ACM-18544

> When you deploy the subscription application, it will display a warning message for the subscription node. If you check the subscription node details, it incorrectly shows the local-cluster is offline. 

Reviewer 2: This issue only affects the application topology page so I would suggest mentioning it to be more specific where the error is occuring. ie.

`When you deploy the subscription application, it will display a warning message for the subscription node in the application topology page. If you check the subscription node details, it incorrectly shows the local-cluster is offline.`

Reviewer 2: To check the real status of local-cluster, users can go to the ACM console Infrastructure > Clusters page

> When you deploy the subscription application, it will display a warning message for the subscription node in the application topology page. If you check the subscription node details, it incorrectly shows the local-cluster is offline. Check the real status of the local-cluster in the ACM console in menu:Infrastructure[Clusters page].

Reviewer: We don't abbreviate the product name.
```suggestion
When you deploy the Subscription application, it displays a warning message for the Subscription node in the _Application Topology_ page. If you check the Subscription node details, it incorrectly shows the `local-cluster` is offline. Check the real status of the `local-cluster` in the {acm-short} console _Clusters_ page.
```

> When you deploy the subscription application, it will display a warning message for the subscription node in the application topology page. If you check the subscription node details, it incorrectly shows the local-cluster is offline. Check the real status of the local-cluster in the ACM console in menu:Infrastructure[Clusters page].

Reviewer 3: ```suggestion
When you deploy the subscription application, it displays a warning for the subscription node in the application topology page. If you check the subscription node details, it incorrectly shows the `local-cluster` as offline.

You can check the correct status of the `local-cluster` by going to *Infrastructure* > *Clusters* in the console.
```
